### DISCUSSION TITLE
Menu cursor spins after playing beatmap

### DISCUSSION THREAD

**Discussion by brynblack** [ppy/osu#16112] [Q&A] [answered]
**Description:**
For some reason when you play a beatmap with a spinner at the end, and you spin for a long enough duration whilst holding down your mouse button or tablet pen, then letting go of your mouse button or tablet pen when the beatmap goes to the results screen, the menu cursor spins very fast. This seems to only happen when the "Rotate cursor when dragging" setting is enabled. I'm not sure if this is intended or not, but I thought I might like to bring it up.

https://user-images.githubusercontent.com/49110391/146348960-c0b0f74c-0f5f-45ac-8682-4b1ee5a08414.mp4

**Logs:**
[runtime.log](https://github.com/ppy/osu/files/7726209/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7726210/updater.log)
[database.log](https://github.com/ppy/osu/files/7726211/database.log)
[network.log](https://github.com/ppy/osu/files/7726213/network.log)
[performance.log](https://github.com/ppy/osu/files/7726214/performance.log)

**Specs:**
Ryzen 5 3600
RTX 2070 Super
G.Skill Trident Z RGB 16GB
Windows 10 Pro
osu! 2021.1211.0-lazer

**Comment by peppy** [comment]
Interesting one. The menu cursor probably shouldn't get wound up from gameplay spinning.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16114
